dts/bindings/sensor/st,iis2iclx-common.yaml

```diff
@@ -20,3 +20,15 @@ properties:
         This number represents which of the two interrupt pins
         (INT1 or INT2) the line is attached to. This property is not
         mandatory and if not present it defaults to 1.
+
+    range:
+      type: int
+      required: false
+      default: 3
+      description: Range in g. Default is power-up configuration.
+      enum:
+        - 0 # 500mg (0.015 mg/LSB)
+        - 1 # 3g    (0.122 mg/LSB)
+        - 2 # 1g    (0.031 mg/LSB)
+        - 3 # 2g    (0.061 mg/LSB)
+
```

Comment:
`git commit` normally complains about empty last lines, not sure why it didn't here.

Reply:
Looks like it's corrected in the next commit.